src/hive/core.cljs

```diff
@@ -9,31 +9,48 @@
             [hive.queries :as queries]
             [hive.rework.util :as tool]
             [hive.components.screens.home.core :as home]
+            [hive.components.screens.errors :as errors]
             [hive.components.router :as router]
             [hive.components.screens.settings.core :as settings]
             [hive.components.screens.settings.city-picker :as city-picker]
             [cljs.core.async :as async]
             [cljs-react-navigation.reagent :as rn-nav]
-            [hive.components.screens.home.route :as route]))
+            [hive.components.screens.home.route :as route]
+            [hive.components.foreigns.react :as react]))
 
-"Each Screen will receive two props:
- - screenProps - Extra props passed down from the router (rarely used)
- - navigation  - The main navigation functions in a map as follows:
-   {:state     - routing state for this screen
-    :dispatch  - generic dispatch fn
-    :goBack    - pop's the current screen off the stack
-    :navigate  - most common way to navigate to the next screen
-    :setParams - used to change the params for the current screen}"
+(defn- screenify
+  [component title]
```

Comment:
@mehdisadeghi title is just one of many navigation options that we can provide. `screenify` should not artifitially limit those to only `title`. See [this link](https://reactnavigation.org/docs/en/stack-navigator.html#navigationoptions-for-screens-inside-of-the-navigator) for info on the possible flags.

I think a simple map of `options` would be enough

Reply:
PS: I think that you can even just leave those options out for the time being since we are not using them so it might be confusing to think that they are required 😅